### ISSUE TITLE
 Session ID failed to authorize your account. Try refreshing your session id. (new)

### ISSUE THREAD

**Issue by CastOnCode** [currency-cop/currency-cop#156]
Opening identic issue to already existing, https://github.com/currency-cop/currency-cop/issues/148 because it came back.
![image](https://user-images.githubusercontent.com/60432009/97630399-371bed80-1a38-11eb-814b-6bdabfd41fec.png)

Is it coming from the same cause or? 
Obviously, I've already refreshed and relogined and refreshed POESESSID

**Comment by helioborges** [comment]
Yeah me too, since yesterday. I even installed the new beta version, the same error

**Comment by gotex007** [comment]
Been like this for at least two weeks for me.

**Comment by slypez** [comment]
Same for me, any fix coming or is there an alternative to this?

**Comment by blackmorekwb** [comment]
still encountering this myself.

**Comment by blackmorekwb** [comment]
> still encountering this myself.

i uninstalled the program, and deleted the currency cop folder in the appdata directory as well. reinstalled the latest release; still got error. logged out of the poe website, logged in, refreshed the cookie values -> still error. 

i repeated and installed the second latest release, and alas i am still not getting it to work. I know the session id is good, i am even using it with another POE third party program. is there anything else that i can try?

**Comment by helioborges** [comment]
I think this is dead. 

Using Exilence and it's great.
